crucible-mir-comp/src/Mir/Compositional/Clobber.hs

```diff
@@ -51,17 +51,18 @@ clobberSymbolic sym loc nameStr shp0 rv0 = go shp0 rv0
         MirVector_Vector v -> MirVector_Vector <$> mapM (go shp) v
         MirVector_PartialVector pv ->
             MirVector_PartialVector <$> mapM (mapM (go shp)) pv
-        MirVector_Array _ -> error $ "clobberSymbolic: MirVector_Array is unsupported"
+        MirVector_Array _ -> error $
+          "clobberSymbolic: MirVector_Array is unsupported, for " ++ nameStr
```

Comment:
It looks like every `error` call here prints a message followed by "for <nameStr>". It might be worth factoring this out into a locally defined function. (Similarly for `clobberImmutSymbolic`.)